### PR TITLE
Check  grammar and word spelling errors in docs

### PR DESCRIPTION
**Pull request by Pikaqiu1999**

### CODE REVIEW

**Comment by Pikaqiu1999** [comment]
I have read the CLA Document and I hereby sign the CLA

File: docs/developer/algorithm/federated_learning/fed_stc.md [hunk 1]
```diff
@@ -9,7 +9,7 @@
...
 1. Compared with the previous sparse work on upstream (client 2 server), FedSTC also sparses on downstream (server 2 client);
 2. When only some clients participate in each round, a Weight Update Caching mechanism is provided on the server side. Each client must synchronize the latest model before participating in the next round of training, or lag behind global weights. updates; (I understand such motivation is that if only part of the updates are updated, the content to be transmitted can be sparse);
-3. Quantization is added while sparse. The quantization method is Binarizaiton. Only 3 numbers will appear in the final matrix, {![](https://latex.codecogs.com/svg.image?\inline&space;\small&space;\mu), 0, ![](https://latex.codecogs.com/svg.image?\inline&space;\small&space;\mu)};
+3. Quantization is added while sparse. The quantization method is Binarizaton. Only 3 numbers will appear in the final matrix, {![](https://latex.codecogs.com/svg.image?\inline&space;\small&space;\mu), 0, ![](https://latex.codecogs.com/svg.image?\inline&space;\small&space;\mu)};
 4. Lossless Golomb Encoding is used on the sparse + quantized matrix;
```
Comment (tpppppub) [inline docs/developer/algorithm/federated_learning/fed_stc.md:13]: `Binarizaton` -> `Binarization`

Comment (Pikaqiu1999) [reply docs/developer/algorithm/federated_learning/fed_stc.md:13]: > `Binarizaton` -> `Binarization`

已修改

**Review by tpppppub — APPROVED** [review]
LGTM and THX.

**Comment by tpppppub** [comment]
hi @Pikaqiu1999 , @lsw666-gif  is the committer, he/her has to sign the CLA.

**Comment by lsw666-gif** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by lsw666-gif** [comment]
> hi @Pikaqiu1999 , @lsw666-gif is the committer, he/her has to sign the CLA.

收到

**Pull request merged**